frontend/src/containers/Homepage/Homepage.js

```diff
@@ -41,7 +41,7 @@ class Homepage extends Component {
               <div className="col-12">
                 <h2>Our Suggestions</h2>
                 <br />
-                <h6>{ 'Based on this week\'s popular ingredient donations' }</h6>
+                <h5>{ 'Based on this week\'s popular ingredient donations' }</h5>
```

Comment:
NitPick: Will this work without the curly braces? I don't think those are necessary here.

Reply:
Either way, you could use `&apos;` in place of the straight single quote, which would be more typographically correct anyway.